design/snapshots.md

```diff
@@ -21,7 +21,7 @@ minimal API simplifies behavior without sacrificing power. This makes the
 surface area for driver implementations smaller, ensuring that behavior is more
 consistent between implementations.
 
-These differ from the concept of the graphdriver in that the _Snapshot Manager_
+These differ from the concept of the graphdriver in that the _Snapshotter_
 has no knowledge of images or containers. Users simply prepare and commit
 directories. We also avoid the integration between graph drivers and the tar
 format used to represent the changesets.
```

Comment:
Might be good to include a link the godoc for the snapshotter so that people can link the documentation to the implementation.